src/pages/MainPage/MainPage.jsx

```diff
@@ -1,24 +1,49 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import Container from "react-bootstrap/Container";
 import Button from "react-bootstrap/Button";
 
-import { ADD_TO_BASKET } from "../../redux/store/actions";
+import Basket from "../Basket";
+
 import style from "./MainPage.module.scss";
+import { fetchUsers } from "../../services/Servise";
+import Skeleton from "../../Ui/Skeleton";
+const imgPerson =
+  "https://previews.123rf.com/images/fokaspokas/fokaspokas1806/fokaspokas180600645/103145234-team-few-person-white-icon-with-shadow-on-transparent-background.jpg";
 
 const MainPage = () => {
   const dispatch = useDispatch();
-  const products = useSelector((state) => state.items.it);
+  const users = useSelector((state) => state.items.users);
+  const [showSkeleton, setShowSkeleton] = useState(true);
+  const usersss = users[0];
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      dispatch(fetchUsers());
+    }, 2000);
+    return () => {
+      clearTimeout(timer);
```

Comment:
why u clear timeout on unmont?

Reply:
because i need to complete my timer working ,this is my mind!